### ISSUE TITLE
UI Navsat and Odom Appear Mirrored

### ISSUE THREAD

**Issue by mike-matera** [BCLab-UNM/Swarmathon-ROS#45]
Hi UNM Team,

I was testing out the new release this morning and I noticed that the odometry and the navsat paths in the UI map seem to be mirrors of eachother. Seems like that might cause some problems! 

Cheers
./m

**Comment by JCHJones** [comment]
Hi mike my name is Jarett and I work on the UNM team. In a perfect world with no error or noise the odometry, navsat, and map frames would perfectly mirror each other at all times. Deviations from each other will occur over time as things like drift from encoders causes odometry to become inaccurate and GPS noise causes navsat and map to jump around. In sim they will stay pretty close at all times beacuse error is less pronounced and difficult to properly simulate compared to physical robots. This is one of many reasons we tell physical teams that the sim is only so good for testing and that what works in sim often will not on a physical robot.

**Comment by JCHJones** [comment]
However if you mean they appear to be mirrored as in opposite sides of an axis this is not good and would cause problems. Please let us know if this is sim or physical

**Comment by gmfricke** [comment]
Hello, 

thanks for reporting this. I have created a message in the software bugs section of the forum: [Forum post](https://swarmathon.discussion.community/post/show_single_post?pid=1294516245&postcount=2&forum=488537).

**Comment by mike-matera** [comment]
Hi Matthew, 

Sorry, I misremembered the bug report protocol! I'll cross-post this to the forum. 

@JCHJones The problem is not normal drift. When GPS says the rover is in the North-West corner of the arena odometry seems to say it's in the South-East. I noticed it when I selected all checkmarks in the UI and saw my rover's path looked like a butterfly. One wing was GPS and the other was odometry. Pretty, but not good. 

Cheers
./m
